Linking curl broken since roughly 2 days ago

### Description / Steps to reproduce the issue

Curl is failing to statically link since ~2 days ago.

1. Install `mingw-w64-ucrt-x86_64-curl`
2. Find curl like: https://github.com/LizardByte/Sunshine/blob/f0a00ae35683dfd5742952fb4cf99edf268b57fd/cmake/dependencies/common.cmake#L16
3. Statically link curl like: https://github.com/LizardByte/Sunshine/blob/f0a00ae35683dfd5742952fb4cf99edf268b57fd/cmake/compile_definitions/windows.cmake#L15-L18
4. Include curl like: https://github.com/LizardByte/Sunshine/blob/f0a00ae35683dfd5742952fb4cf99edf268b57fd/cmake/compile_definitions/common.cmake#L140
5. Final link: https://github.com/LizardByte/Sunshine/blob/f0a00ae35683dfd5742952fb4cf99edf268b57fd/cmake/targets/common.cmake#L25

### Expected behavior

Curl properly links as it did previously.

### Actual behavior

The initial error (~2 days ago) was many `undefined references` (https://github.com/LizardByte/Sunshine/actions/runs/9278791852/job/25530652407?pr=2604#step:7:501)

```txt
[ 81%] Linking CXX executable sunshine.exe
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x430): undefined reference to `__imp_nghttp3_conn_read_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x5a2): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x69a): undefined reference to `__imp_nghttp3_conn_close_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x73a): undefined reference to `__imp_nghttp3_conn_close_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x75c): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x880): undefined reference to `__imp_nghttp3_conn_resume_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x95d): undefined reference to `__imp_nghttp3_conn_shutdown_stream_read'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x973): undefined reference to `__imp_nghttp3_conn_close_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x986): undefined reference to `__imp_nghttp3_conn_set_stream_user_data'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0xe4d): undefined reference to `__imp_nghttp3_rcbuf_get_buf'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x1fe0): undefined reference to `__imp_nghttp3_conn_writev_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x228d): undefined reference to `__imp_nghttp3_conn_block_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x2335): undefined reference to `__imp_nghttp3_conn_add_write_offset'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x235f): undefined reference to `__imp_nghttp3_conn_add_ack_offset'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x2597): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x25d3): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x25ef): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x2721): undefined reference to `__imp_nghttp3_conn_unblock_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x29e1): undefined reference to `__imp_nghttp3_conn_resume_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x2f44): undefined reference to `__imp_nghttp3_conn_del'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x3848): undefined reference to `__imp_nghttp3_conn_submit_request'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x39db): undefined reference to `__imp_nghttp3_conn_resume_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x3deb): undefined reference to `__imp_nghttp3_strerror'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4a19): undefined reference to `__imp_nghttp3_settings_default_versioned'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4a1f): undefined reference to `__imp_nghttp3_mem_default'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4a50): undefined reference to `__imp_nghttp3_conn_client_new_versioned'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4ba9): undefined reference to `__imp_nghttp3_conn_bind_control_stream'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4bd1): undefined reference to `__imp_nghttp3_conn_bind_qpack_streams'
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-curl_osslq.o):(.text+0x4fb0): undefined reference to `__imp_nghttp3_version'
collect2.exe: error: ld returned 1 exit status
mingw32-make[2]: *** [CMakeFiles\sunshine.dir\build.make:836: sunshine.exe] Error 1
mingw32-make[1]: *** [CMakeFiles\Makefile2:214: CMakeFiles/sunshine.dir/all] Error 2
mingw32-make[1]: *** Waiting for unfinished jobs....
```

But now it is many `multiple definition` (https://github.com/LizardByte/Sunshine/actions/runs/9278791852/job/25604282363?pr=2604#step:7:504)

```
[ 84%] Linking CXX executable sunshine.exe
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3e80): multiple definition of `get_dll_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0x0): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3e850): multiple definition of `sanitise_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0x190): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3e8d0): multiple definition of `simplify_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0x210): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3ebf0): multiple definition of `get_relative_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0x510): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f110): multiple definition of `get_executable_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0x9b0): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f1f0): multiple definition of `strip_n_suffix_folders'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xa70): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f240): multiple definition of `get_relocated_path_list'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xac0): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f2b0): multiple definition of `get_relocated_path_list_lib'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xb00): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f370): multiple definition of `single_path_relocation'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xb90): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f440): multiple definition of `single_path_relocation_lib'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xca0): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f560): multiple definition of `msys2_get_relocated_single_path'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xdf0): first defined here
D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0x3f6d0): multiple definition of `msys2_get_relocated_path_list'; D:/a/_temp/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcrypto.a(libcrypto-lib-pathtools.obj):(.text+0xf20): first defined here
collect2.exe: error: ld returned 1 exit status
mingw32-make[2]: *** [CMakeFiles\sunshine.dir\build.make:836: sunshine.exe] Error 1
mingw32-make[1]: *** [CMakeFiles\Makefile2:214: CMakeFiles/sunshine.dir/all] Error 2
mingw32-make[1]: *** Waiting for unfinished jobs....
```

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

Microsoft Windows Server 2019, 10.0.17763 (GitHub hosted runner)

### MINGW environments affected

- [ ] MINGW64
- [ ] MINGW32
- [X] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

### Are you willing to submit a PR?

I would if I knew what the fix was.

--- a/mingw-w64-curl/0004-more-static-fixes.patch
+++ b/mingw-w64-curl/0004-more-static-fixes.patch
@@ -4,7 +4,7 @@
  URL: https://curl.se/
  Description: Library to transfer files with ftp, http, etc.
  Version: @CURLVERSION@
-+Requires.private: libidn2 libbrotlidec
++Requires.private: libidn2 libbrotlidec libnghttp2 libnghttp3
  Libs: -L${libdir} -lcurl @LIBCURL_NO_SHARED@
  Libs.private: @LIBCURL_LIBS@
  Cflags: -I${includedir} @CPPFLAG_CURL_STATICLIB@

--- a/mingw-w64-curl/PKGBUILD
+++ b/mingw-w64-curl/PKGBUILD
@@ -7,7 +7,7 @@ pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
            "${MINGW_PACKAGE_PREFIX}-${_realname}-gnutls" \
            "${MINGW_PACKAGE_PREFIX}-${_realname}-winssl"))
 pkgver=8.8.0
-pkgrel=7
+pkgrel=8
 pkgdesc="Command line tool and library for transferring data with URLs (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
@@ -54,7 +54,7 @@ sha256sums=('0f58bb95fc330c8a46eeb3df5701b0d90c9d9bfcc42bd1cd08791d12551d4400'
             '1585ef1b61cf53a2ca27049c11d49e0834683dfda798f03547761375df482a90'
             'adfa29d6f427fc6154c90e6ce297879ef88dc1831a7a04497d803acf3de93eea'
             'f411ae7662fc67ff52bdc7d42a92e255e52154bb7d06251694802ba8d709319d'
-            'e1f32a6b03e337f188262b6201980c874138f0524123afb11285d0802451e5b1')
+            '9f7394beffb48e36e282fab03762a7c1853fcb245f94c79f7a93607c1b243515')
 validpgpkeys=('27EDEAF22F3ABCEB50DB9A125CC908FDB71E12C2')  # Daniel Stenberg
 
 apply_patch_with_msg() {

--- a/mingw-w64-nghttp2/002-nghttp2-pkgconfig-add-cflags-private.patch
+++ b/mingw-w64-nghttp2/002-nghttp2-pkgconfig-add-cflags-private.patch
@@ -1,0 +1,7 @@
+--- a/lib/libnghttp2.pc.in
++++ b/lib/libnghttp2.pc.in
+@@ -31,3 +31,4 @@
+ Version: @VERSION@
+ Libs: -L${libdir} -lnghttp2
+ Cflags: -I${includedir}
++Cflags.private: -DNGHTTP2_STATICLIB

--- a/mingw-w64-nghttp2/PKGBUILD
+++ b/mingw-w64-nghttp2/PKGBUILD
@@ -4,7 +4,7 @@ _realname=nghttp2
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
 pkgver=1.61.0
-pkgrel=1
+pkgrel=2
 pkgdesc="Framing layer of HTTP/2 is implemented as a reusable C library (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
@@ -22,14 +22,17 @@ makedepends=(
 )
 license=('spdx:MIT')
 source=("https://github.com/nghttp2/nghttp2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz"
-        "001-fix-pc-prefix.patch")
+        "001-fix-pc-prefix.patch"
+        "002-nghttp2-pkgconfig-add-cflags-private.patch")
 sha256sums=('c0e660175b9dc429f11d25b9507a834fb752eea9135ab420bb7cb7e9dbcc9654'
-            'baf901e7d565eaea1133d30b13620b02681ce96ebe56d38e1dea4394299106b6')
+            'baf901e7d565eaea1133d30b13620b02681ce96ebe56d38e1dea4394299106b6'
+            'd82d0b5b32e26955c99ef70e129b33696fee01ad9a9a4e5d0fb98951fb5d0f22')
 
 prepare() {
   cd $srcdir/${_realname}-${pkgver}
 
   patch -Nbp1 -i "${srcdir}/001-fix-pc-prefix.patch"
+  patch -Nbp1 -i "${srcdir}/002-nghttp2-pkgconfig-add-cflags-private.patch"
 }
 
 build() {

--- a/mingw-w64-nghttp3/002-nghttp3-pkgconfig-add-cflags-private.patch
+++ b/mingw-w64-nghttp3/002-nghttp3-pkgconfig-add-cflags-private.patch
@@ -1,0 +1,7 @@
+--- a/lib/libnghttp3.pc.in
++++ b/lib/libnghttp3.pc.in
+@@ -32,3 +32,4 @@
+ Version: @VERSION@
+ Libs: -L${libdir} -lnghttp3
+ Cflags: -I${includedir}
++Cflags.private: -DNGHTTP3_STATICLIB

--- a/mingw-w64-nghttp3/PKGBUILD
+++ b/mingw-w64-nghttp3/PKGBUILD
@@ -4,7 +4,7 @@ _realname=nghttp3
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
 pkgver=1.3.0
-pkgrel=1
+pkgrel=2
 pkgdesc="HTTP/3 library written in C (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
@@ -19,16 +19,19 @@ makedepends=(
   "${MINGW_PACKAGE_PREFIX}-ninja"
 )
 source=("https://github.com/ngtcp2/nghttp3/releases/download/v${pkgver}/nghttp3-${pkgver}.tar.xz"{,.asc}
-        "001-fix-pc-prefix.patch")
+        "001-fix-pc-prefix.patch"
+        "002-nghttp3-pkgconfig-add-cflags-private.patch")
 sha256sums=('450525981d302f23832b18edd1a62cf58019392ca6402408d0eb1a7f3fd92ecf'
             'SKIP'
-            '08559edf597300d34b258c5dab6d16237c6493976e7b4d45a818e0619d21665d')
+            '08559edf597300d34b258c5dab6d16237c6493976e7b4d45a818e0619d21665d'
+            '68c831018eb79bc3b43a8fb6a7d013a617dbe2ba7d7971b0993b3cf27107e614')
 validpgpkeys=('F4F3B91474D1EB29889BD0EF7E8403D5D673C366') # Tatsuhiro Tsujikawa <tatsuhiro.t@gmail.com>
 
 prepare() {
   cd "${_realname}-${pkgver}"
 
   patch -Np1 -i "$srcdir/001-fix-pc-prefix.patch"
+  patch -Np1 -i "$srcdir/002-nghttp3-pkgconfig-add-cflags-private.patch"
 }
 
 build() {

